server/db/categories.js

```diff
@@ -0,0 +1,16 @@
+const environment = process.env.NODE_ENV || 'development'
+const config = require('./knexfile')[environment]
+const knex = require('knex')(config)
+
+function getCategoriesAndSubcategories (db = knex) {
+  return db('categories')
+    .join('subcategories', 'categories.id', '=', 'subcategories.category_id')
+    .select(
+      'categories.name as categoryName',
+      'subcategories.name as subcategoryName'
+    )
```

Comment:
Maybe add a getSubcategoriesByCategory (categoryId) that returns an array of subcategories?